sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/aio/_text_analytics_client_async.py

```diff
@@ -789,6 +790,10 @@ async def begin_analyze_batch_actions(  # type: ignore
                 key_phrase_extraction_tasks=[
                     t.to_generated() for t in
                     [a for a in actions if _determine_action_type(a) == AnalyzeBatchActionsType.EXTRACT_KEY_PHRASES]
+                ],
+                entity_linking_tasks=[
+                    t.to_generated() for t in
+                    [a for a in actions if _determine_action_type(a) == AnalyzeBatchActionsType.RECOGNIZE_LINKED_ENTITIES]
```

Comment:
can you add some tests for linked entities and update the samples (readme and actual samples)? thanks!

Reply:
Yeah, I was working on this, just ran out of time yesterday.

Reply:
Ok, I've updated tests and samples.  I've deleted a few recordings in preparation for running them, but I don't think this has been deployed to WestUS2 yet so I will wait to re-run the tests for now.  In the meantime let me know if there are any other tests I should modify/add.